### PR TITLE
Add checksum for py-prettytable@3.2.0

### PR DESCRIPTION
**Pull request by aandvalenzuela**

### CODE REVIEW

**Review by adamjstewart — CHANGES_REQUESTED** [review]
Python 3.7+ is now required

File: var/spack/repos/builtin/packages/py-more-itertools/package.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ class PyMoreItertools(PythonPackage):
     homepage = "https://github.com/erikrose/more-itertools"
     pypi = "more-itertools/more-itertools-7.2.0.tar.gz"
 
+    version('8.12.0', sha256='7dc6ad46f05f545f900dd59e8dfb4e84a4827b97b3cfecb175ea0c7d247f6064')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-more-itertools/package.py]: I think these changes got added to the wrong PR

**Comment by adamjstewart** [comment]
This PR still conflicts with #30325. Can you remove those changes or rebase the PR?

**Comment by aandvalenzuela** [comment]
My bad, sorry! Now it should be solved :)

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**